### PR TITLE
add longzhu

### PR DESCRIPTION
**Pull request by longzhudev**
add longzhu

### CODE REVIEW

**Review by e-asia — COMMENTED** [review]
@longzhudev 
- The address and token parameters seem to be missing
- The css selector should be longzhu instead of yam

File: spaces/longzhu/index.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "key": "longzhu",
+  "symbol": "LONGZHU",
+  "name": "Longzhu Finance",
+  "defaultView": "core",
+  "address": "",
+  "token": "",
```
Comment (e-asia) [inline spaces/longzhu/index.json:7]: The address and token parameters seem to be missing

File: spaces/longzhu/skin.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.yam {
```
Comment (e-asia) [inline spaces/longzhu/skin.scss:1]: The css selector should be longzhu instead of yam

**Comment by bonustrack** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**